core/templates/dev/head/components/entity-creation-services/topic-creation.service.ts

```diff
@@ -55,26 +63,28 @@ angular.module('oppia').factory('TopicCreationService', [
           ]
         });
 
-        modalInstance.result.then(function(topicName) {
-          if (topicName === '') {
+        modalInstance.result.then(function(topic) {
+          if (topic.topicName === '') {
```

Comment:
I don't see a warning here for empty abbreviated topic name?

Reply:
Done.